worlds/mmx3/__init__.py

```diff
@@ -327,39 +389,57 @@ def fill_slot_data(self):
 
 
     def generate_early(self):
+        # Parse logic-relevant settings and save them elsewhere (needed for UT support)
+        self.boss_weakness_strictness = self.options.boss_weakness_strictness.value
+        self.pickupsanity = self.options.pickupsanity.value
+        self.jammed_buster = self.options.jammed_buster.value
+        self.logic_boss_weakness = self.options.logic_boss_weakness.value
```

Comment:
Pycharm is probably screaming about these. I'd really recommend against this pattern

Reply:
Take a look at how TUNIC is doing it -- we're setting the option values directly based on what was in the slot_data

Reply:
Oh, that looks much better. Assuming I'm looking at [the correct thing](https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/tunic/__init__.py#L106-L124)

Reply:
Yup! That's the right thing there. Some of the options get set a little differently -- you can just ignore those, they're because of ER stuff and how I'm handling it.